### PR TITLE
Add blackstone recipes to Grindstone and Ore Crusher

### PR DESCRIPTION
**Pull request by Phoenix-Starlight**
## Description

I am adding blackstone recipes to complete the remaining machines that do not have an appropriate blackstone recipe.
## Proposed changes

Added recipes to Grindstone and Ore Crusher
## Checklist

- [ ] I have fully tested the proposed changes and promise that they will not break everything into chaos.
- [ ] I have also tested the proposed changes in combination with various popular addons and can confirm my changes do not break them.
- [x] I followed the existing code standards and didn't mess up the formatting.
- [x] I have made sure that the proposed changes do not break compatibility across the supported Minecraft versions (1.16.* - 1.20.*).
- [ ] I did my best to add documentation to any public classes or methods I added.
- [ ] I have added `Nonnull` and `Nullable` annotations to my methods to indicate their behaviour for null values
- [ ] I added sufficient Unit Tests to cover my code.

### CODE REVIEW

**Review by JustAHuman-xD — COMMENTED** [review]
Just a tiny thing but maybe put it after the other gravel recipes? For consistency, could even move the deepslate one too. Totally not required hence why this is just a comment

**Comment by Phoenix-Starlight** [comment]
I decided to just alphabetize the variants. (Although really, I'd rather alphabetize the entire list)

**Comment by Phoenix-Starlight** [comment]
Cobbled deepslate cant be moved unless I move the entire version check.

**Comment by JustAHuman-xD** [comment]
> Cobbled deepslate cant be moved unless I move the entire version check.

Yeah you would have to move the check & flip the order or duplicate the check which wouldn't be ideal

File: src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java [hunk 1]
```diff
@@ -50,6 +50,9 @@ public OreCrusher(ItemGroup itemGroup, SlimefunItemStack item) {
 
     @Override
     protected void registerDefaultRecipes(List<ItemStack> recipes) {
+        recipes.add(new ItemStack(Material.BLACKSTONE, 8));
+        recipes.add(new ItemStack(Material.RED_SAND, 1));
```
Comment (JustAHuman-xD) [inline src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java:54]: Are we sure that this is super logical? The cobblestone to sand makes sense to me but imo blackstone to red sand does not. Could just be me though

Comment (Sefiraat) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java:54]: While the relation ship from Blackstone > Sand isn't the best, it IS nice to have some variation in output, and add a way to renew a new type of... boring(?) item.

Comment (Phoenix-Starlight) [reply src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/multiblocks/OreCrusher.java:54]: I was doing it for equivalency, really.

**Review by JustAHuman-xD — APPROVED** [review]
LGTM, if anyone thinks of anything else for the red sand we can change it later tbf

**Pull request merged**